modules/packer/packer.go

```diff
@@ -19,6 +21,38 @@ type Options struct {
 	Env      map[string]string // Custom environment variables to set when running Packer
 }
 
+// BuildArtifactsE can take a map of identifierName <-> Options and then parallelize
+// the packer builds. Once all the packer builds have completed a map of identifierName <-> generated identifier
+// is returned. If any artifact fails to build, the errors are accumulated and returned
+// as a MultiError. The identifierName can be anything you want, it is only used so that you can
+// know which generated artifact is which.
+func BuildArtifactsE(t *testing.T, artifactNameToOptions map[string]*Options) (map[string]string, error) {
+	var waitForArtifacts sync.WaitGroup
+	waitForArtifacts.Add(len(artifactNameToOptions))
+
+	var artifactNameToArtifactId = map[string]string{}
+	errorsOccurred := []error{}
+
+	for artifactName, curOptions := range artifactNameToOptions {
+		artifactName := artifactName
+		curOptions := curOptions
```

Comment:
Add comment explaining why you need this capture

Reply:
done